django-stubs/db/models/base.pyi

```diff
@@ -7,6 +7,13 @@ from django.db.models.options import Options
 
 _Self = TypeVar("_Self", bound="Model")
 
+class ModelStateFieldsCacheDescriptor: ...
+
+class ModelState:
+    db: None = ...
```

Comment:
Is this type correct? Looks like we are missing its other value. And it has to be `Optional[...]`

Reply:
It should be `Optional[str]`